### DISCUSSION TITLE
Is the branch pinocchio3-preview compilent with hpp-fcl 2.4.4？

### DISCUSSION THREAD

**Discussion by dogged1021** [stack-of-tasks/pinocchio#2208] [Q&A] [answered]
Hi, I want to use the latest version of HPP-FCL in Pinocchio, but I am confused about [#1620]. Can the branch pinocchio 3-preview compile with hpp-fcl 2.4.4? Or which version of HPP-FCL can compile with the latest release version of pinocchio-2.x?

Thanks for your time!

OS: Ubuntu 20.04 LTS
Program language: C++

**Comment by jcarpent** [comment, marked as answer]
Yes, it should work perfectly.
